### PR TITLE
resolve #13 reorganize sections

### LINKED ISSUE #13: Reorganize Constitution Sections
Check one:
- [X] Semantic Issue: something about the meaning of the text is different
- [x] Non-semantic Issue: Spelling, grammar, or formatting changes.

Summary of issue:

needs discussion

the early draft included the following sections:

- Introduction
- Structure
- Membership
- Officers of the House
- General Operations of the House
- Voting
- Modifications to the Constitution
- Judicial
- Adherence to University Policy

This included three major refactors:

1 - moving Packet and Evaluations into the Membership section
2 - moving operations of the Evaluations/Financial directorships into "General Operations of the House"
3 - moving maintainers and RTPs under "Officers of the House"

this has the benefit of increasing the cohesion of each section, so that each section is more closely related to a single task

### PR DESCRIPTION
**Pull request by rhochgraf21**
Check one:
- [ ] Semantic Change: something about the meaning of the text is different
- [X] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):

1 - moving Packet, Introductory Evaluations, Membership Evaluations, and Selection Processes into the Membership section
2 - moving operations of the Evaluations/Financial directorships into "General Operations of the House"
3 - moving maintainers and RTPs under "Officers of the House"

### CODE REVIEW

**Comment by mxmeinhold** [comment]
this changed a committed pdf? I don't see the source here

**Comment by rhochgraf21** [comment]
made a silly goof. should be fixed now

**Review by esoccoli — APPROVED** [review]
lgtm

**Comment by rhochgraf21** [comment]
pinging @Sam-Cordry since you asked for ping

**Review by IgorPolotai — APPROVED** [review]
lgtm

**Comment by esoccoli** [comment]
@Sam-Cordry

File: constitution.tex [hunk 1]
```diff
@@ -866,12 +656,193 @@
...
+Members may sign a more recent revision of the Code of Conduct to update their agreement.
+\asubsubsubsection{Changes}
+The following process is used for making changes to the Code of Conduct document.
+\begin{enumerate}
+	\item A Root Type Person drafts a change to the Code of Conduct and makes publicly available both the summary and difference file of the change.
+	\item The change is proposed at a House Meeting and is discussed at the same House Meeting.
+	\item The final proposal is presented at the next House Meeting and ballots are distributed for a Balloted House vote as described in \ref{Balloted Vote}.
+\end{enumerate}
+
+% ARTICLE III - GENERAL OPERATIONS OF THE HOUSE
```
Comment (mxmeinhold) [inline constitution.tex:739]: Do these comments still align?

File: constitution.tex [hunk 2]
```diff
@@ -163,13 +120,97 @@
 \asubsection{Introductory Membership Selection}
 Applicants notify the House of their interest in membership by submitting an application to the Evaluations Director.
 They must then undergo the selection process as defined in \ref{Selection Processes}.
+
+\asubsubsection{Selection Processes}
+\renewcommand{\theenumi}{\alph{enumi}} % For this section, we want items to use letters
```
Comment (mxmeinhold) [inline constitution.tex:125]: Do you ever unset this?

File: constitution.tex [hunk 3]
```diff
@@ -866,12 +656,193 @@
...
+\begin{enumerate}
+	\item A Root Type Person drafts a change to the Code of Conduct and makes publicly available both the summary and difference file of the change.
+	\item The change is proposed at a House Meeting and is discussed at the same House Meeting.
+	\item The final proposal is presented at the next House Meeting and ballots are distributed for a Balloted House vote as described in \ref{Balloted Vote}.
+\end{enumerate}
+
+% ARTICLE III - GENERAL OPERATIONS OF THE HOUSE
+\article{General Operations of the House}
+
+% BY-LAW IV - OPERATIONS OF THE EVALUATIONS DIRECTORSHIP
```
Comment (mxmeinhold) [inline constitution.tex:742]: Mostly asked because this says by law

**Review by Sam-Cordry — APPROVED** [review]
Looks good to me.

File: constitution.tex (hunk 3 above)
Comment (Sam-Cordry) [reply constitution.tex:792]: Good catch, and I propose that we address these issues in #28 and make these changes (and others like it) all in one fell swoop.

**Pull request merged**